browser/resources/settings/brave_appearance_page/brave_appearance_page.html

```diff
@@ -6,7 +6,7 @@
 <link rel="import" href="../settings_shared_css.html">
 <link rel="import" href="../settings_vars_css.html">
 
-<dom-module id="settings-brave-appearance-page">
+<dom-module id="settings-brave-appearance-theme">
```

Comment:
Changes since we want to insert items to two separate places within the chromium list